index.html

```diff
@@ -0,0 +1,233 @@
+                  />Customer Incentives</label
+                >
+                <label
+                  ><input
+                    name="prefer"
+                    value="open-source-community"
+                    type="checkbox"
+                    class="input-checkbox"
+                  />Open Source Marketing Community</label
+                >
+                <label
+                  ><input
+                    name="prefer"
+                    value="gitter-help-rooms"
+                    type="checkbox"
+                    class="input-checkbox"
+                  />Gitter help rooms</label
+                >
+                <label
+                  ><input
+                    name="prefer"
+                    value="videos"
+                    type="checkbox"
+                    class="input-checkbox"
+                  />Videos</label
+                >
+                <label
+                  ><input
+                    name="prefer"
+                    value="city-meetups"
+                    type="checkbox"
+                    class="input-checkbox"
+                  />City Meetups</label
+                >
+                <label
+                  ><input
+                    name="prefer"
+                    value="wiki"
+                    type="checkbox"
+                    class="input-checkbox"
+                  />Wiki</label
+                >
+                <label
+                  ><input
+                    name="prefer"
+                    value="forum"
+                    type="checkbox"
+                    class="input-checkbox"
+                  />Forum</label
+                >
+                <label
+                  ><input
+                    name="prefer"
+                    value="additional-courses"
+                    type="checkbox"
+                    class="input-checkbox"
+                  />Additional Marketing Courses</label
+                >
+              </div>
+          
+              <div class="form-group">
+                <p>Let know what's on your Mind</p>
+                <textarea
+                  id="comments"
+                  class="input-textarea"
+                  name="comment"
+                  placeholder="Enter your comment here..."
+                ></textarea>
+              </div>
+          
+              <div class="form-group">
+                <button type="submit" id="submit" class="submit-button">
+                  Submit
+                </button>
+              </div>
+            </form>
+          </div>
+    </main>
+    
+
+    <!-- <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script> -->
+
+    <!--
+    Hello Camper!
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+    - The freeCodeCamp Team
+  
+    -->
```

Comment:
I think it would be better to remove the comments in order to have a clean code.